src/melee/gr/grbattle.c

```diff
@@ -315,9 +342,13 @@ static void func_8021A3BC(HSD_GObj* gobj)
     HSD_JObj* jobj = gobj->hsd_obj;
     HSD_GObj* bg_gobj;
     s32 temp_r0_2;
-    s16 sp28[BATTLE_BG_MAX] = { 1, 2, 4 };
+    s16 sp28[] = { 1, 2, 4 };
```

Comment:
```suggestion
    s16 sp28[BATTLE_BG_MAX] = { 1, 2, 4 };
```

Reply:
Just thought we should keep this one, since it makes the iteration a little more clear.

Reply:
`clang` complains and I don't think it's worth turning off the relevant warning (`-Wgnu-folding-constant`). Could branch the preprocessor but... not for this, right?

Reply:
Not familiar with that warning, reading the docs and not seeing why it would triggered here. BATTLE_BG_MAX is a constant so why is it complaining?

Reply:
Because it's `const` and not a define. I'll change that and it'll work on both compilers.